From cfbb079ee81eaf50bb173909cbc27dc2f8e2faf8 Mon Sep 17 00:00:00 2001
From: Daniel Brackenbury <daniel.brackenbury@gmail.com>
Date: Fri, 10 Oct 2025 00:44:37 -0400
Subject: [PATCH] remove Nulea M501 usb entry from hwdb

conflict with other mice using same generic dongle identified (e.g. Protoarc EM11)

(cherry picked from commit 4cbfaaa5491c8d4b69ffe79b2b09061ec41fca3d)

Resolves: RHEL-72702
---
 hwdb.d/70-mouse.hwdb | 9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)

diff --git a/hwdb.d/70-mouse.hwdb b/hwdb.d/70-mouse.hwdb
index 38e7d9e1d9..86470bcfda 100644
--- a/hwdb.d/70-mouse.hwdb
+++ b/hwdb.d/70-mouse.hwdb
@@ -825,12 +825,9 @@ mouse:usb:v0e8fp00a7:name:DaKai 2.4G RX:*
 # Nulea
 ##########################################
 
-# Note: it is possible that other devices may use the same wireless dongle,
-# as such this could require revisiting if it causes issues with other mice
-# Nulea M501 Wireless Trackball (USB Receiver)
-mouse:usb:v25a7pfa61:name:Compx 2.4G Receiver Mouse:*
- ID_INPUT_TRACKBALL=1
-
+# Note: The Nulea uses a generic USB dongle. Overriding its value would cause
+# other mice to be erroneously registered as trackballs, so only bluetooth
+# detection is added.
 # Nulea M501 Wireless Trackball (Bluetooth)
 mouse:bluetooth:v000ep3412:name:Nulea BT5.0 Mouse:*
  ID_INPUT_TRACKBALL=1
